### DISCUSSION TITLE
Help download Ios16

### DISCUSSION THREAD

**Discussion by juliegaudet** [community/community#40183] [Code Security]
-

**Comment by jfullstackdev** [comment]
you mean download or update ? 

https://support.apple.com/en-ph/ios/update

as far as I know too, some can be upgraded but you need to check it whether it's allowed by Apple

**Reply by juliegaudet** [reply]
Updated iPhone 12 it cost 1,000$ you would think that I could upgrade Ios16

**Reply by jfullstackdev** [reply]
business wise, it's the latest version, isn't it ? so I would not give the upgrade for the older phones to be upgraded to the latest version because most of the time, they would not buy that latest one anymore

**Reply by juliegaudet** [reply]
I think that's how they get more money from you pay to fix the technical problems then upgrade

**Comment by ossanna16** [comment]
Hi @juliegaudet,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple products. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
